resources/js/Layouts/AppLayout.vue

```diff
@@ -27,6 +27,13 @@
                                 >
                                     Gruppe
                                 </jet-nav-link>
+
+                                <jet-nav-link
+                                    :href="route('app.post.index')"
+                                    :active="'app.post.index' == currentRoute"
+                                >
```

Comment:
We can add a `v-if="$page.props.orga"` for now, no?

Reply:
yes, thanks